base_exception/models/base_exception.py

```diff
@@ -265,6 +265,26 @@ def _compute_exceptions_summary(self):
             else:
                 rec.exceptions_summary = False
 
+    def _popup_exceptions(self):
+        """Used in several dependent modules"""
```

Comment:
Explain also what this method does

Reply:
Well, I added a bit of explanation 😅

The `account_move_exception` uses this method this way:
![Selection_485](https://user-images.githubusercontent.com/25005517/179216349-86561799-cd14-4443-8d67-1780d2ceced3.png)